pombola/core/fixtures/initial_data.json

```diff
@@ -0,0 +1,1952 @@
+            "updated": "2019-05-09T11:53:04.927",
+            "subtitle": "",
+            "end_date": "future",
+            "title": 161,
+            "sorting_start_date_high": "0000-99-99",
+            "organisation": 77,
+            "created": "2013-10-16T12:57:39.346",
+            "sorting_end_date": "future",
+            "note": "",
+            "person": 542,
+            "place": null,
+            "sorting_end_date_high": "future",
+            "sorting_start_date": "0000-00-00",
+            "start_date": null
+        },
+        "model": "core.position",
+        "pk": 42985
+    },
+    {
+        "fields": {
+            "category": "political",
+            "updated": "2019-03-19T16:07:30.657",
+            "subtitle": "",
+            "end_date": "2014-05-06",
+            "title": 161,
+            "sorting_start_date_high": "2009-05-06",
+            "organisation": 70,
+            "created": "2013-10-16T12:57:39.382",
+            "sorting_end_date": "2014-05-06",
+            "note": "",
+            "person": 542,
+            "place": 1,
+            "sorting_end_date_high": "2014-05-06",
+            "sorting_start_date": "2009-05-06",
+            "start_date": "2009-05-06"
+        },
+        "model": "core.position",
+        "pk": 42986
+    },
+    {
+        "fields": {
+            "category": "political",
+            "updated": "2015-01-30T13:00:33.303",
+            "subtitle": "",
+            "end_date": "2014-05-06",
+            "title": 161,
+            "sorting_start_date_high": "0000-99-99",
+            "organisation": 85,
+            "created": "2013-10-16T12:57:39.397",
+            "sorting_end_date": "2014-05-06",
+            "note": "",
+            "person": 542,
+            "place": null,
+            "sorting_end_date_high": "2014-05-06",
+            "sorting_start_date": "0000-00-00",
+            "start_date": null
+        },
+        "model": "core.position",
+        "pk": 42987
+    },
+    {
+        "fields": {
+            "category": "political",
+            "updated": "2019-02-11T15:54:52.394",
+            "subtitle": "",
+            "end_date": "2014-05-06",
+            "title": 161,
+            "sorting_start_date_high": "0000-99-99",
+            "organisation": 85,
+            "created": "2013-10-16T12:57:39.411",
+            "sorting_end_date": "2014-05-06",
+            "note": "",
+            "person": 542,
+            "place": null,
+            "sorting_end_date_high": "2014-05-06",
+            "sorting_start_date": "0000-00-00",
+            "start_date": null
+        },
+        "model": "core.position",
+        "pk": 42988
+    },
+    {
+        "fields": {
+            "category": "political",
+            "updated": "2015-01-30T13:00:33.304",
+            "subtitle": "",
+            "end_date": "2014-05-06",
+            "title": 161,
+            "sorting_start_date_high": "0000-99-99",
+            "organisation": 75,
```

Comment:
Let's make a few more positions for the "National Assembly" and the "National Council of Provinces" organisations since they're the ones we often filter for.